flexmeasures/ui/templates/crud/asset.html

```diff
@@ -236,6 +236,36 @@ <h3>All child assets for {{ asset.name }}</h3>
                 </table>
             </div>
         </div>
+        <div class="row">
+            <div class="copy-url" title="Click to copy URL to clipboard">
+                <script>
+                    function copyUrl() {
+                        if (!window.getSelection) {
+                            alert('Please copy the URL from the location bar.');
+                            return;
+                        }
+                        const dummy = document.createElement('p');
+                        dummy.textContent = window.location.href;
+                        document.body.appendChild(dummy);
+
+                        const range = document.createRange();
+                        range.setStartBefore(dummy);
+                        range.setEndAfter(dummy);
+
+                        const selection = window.getSelection();
+                        // First clear, in case the user already selected some other text
+                        selection.removeAllRanges();
+                        selection.addRange(range);
+
+                        document.execCommand('copy');
+                        document.body.removeChild(dummy);
+                    }
+                </script>
+                <a href="#" onclick="copyUrl()">
+                    <i class="fa fa-link"></i>
```

Comment:
The layout is not working well. The icon is not under but on the right of the plot.
The play button that is usually on the right is now moved under the plots.